apollo-portal/src/main/resources/static/scripts/directive/directive.js

```diff
@@ -81,6 +81,7 @@ directive_module.directive('apollonav',
 
                 UserService.load_user().then(function (result) {
                     scope.userName = result.userId;
+                    scope.preferredUsername = result.name;
```

Comment:
this is undefined when running with the default spring security auth

![image](https://user-images.githubusercontent.com/837658/114297420-1f668c00-9ae3-11eb-8b7d-a22924c73f23.png)